kernel/data-pipeline/scenario/cdc/protocol/src/main/proto/CDCResponseProtocol.proto

```diff
@@ -35,6 +35,10 @@ message CDCResponse {
     ServerGreetingResult server_greeting_result = 3;
     StreamDataResult stream_data_result = 4;
     DataRecordResult data_record_result = 5;
+    LoginResult login_result = 6;
+    StartStreamingResult start_streaming_result = 7;
+    StopStreamingResult stop_streaming_result = 8;
+    DropStreamingResult drop_streaming_result = 9;
```

Comment:
Could we cache request type in client? Just like `request_id`. Then we could get request type in client, and remove these definitions in `response`.